helm-charts/codetrans/README.md

```diff
@@ -31,9 +31,7 @@ helm install codetrans codetrans --set global.HF_TOKEN=${HFTOKEN} --set global.m
 
 ### IMPORTANT NOTE
 
-1. To use model `mistralai/Mistral-7B-Instruct-v0.3`, you should first goto the [huggingface model card](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.3) to apply for the model access first. You need to make sure your huggingface token has at least read access to that model.
-
-2. Make sure your `MODELDIR` exists on the node where your workload is schedueled so you can cache the downloaded model for next time use. Otherwise, set `global.modelUseHostPath` to 'null' if you don't want to cache the model.
+Make sure your `MODELDIR` exists on the node where your workload is scheduled so you can cache the downloaded model for next time use. Otherwise, set `global.modelUseHostPath` to 'null' if you don't want to cache the model.
```

Comment:
I'd like better advice than host path usage promotion & not caching models, but that should probably be in  the main Helm-charts.

Tthere are several other READMEs with similarly problematic text, so fixing that clearly belongs to some other PR:
```
GenAIInfra/helm-charts$ find -name README.md | xargs grep -l "MODELDIR.*exists on the node"
./chatqna/README.md
./searchqna/README.md
./codetrans/README.md
./codegen/README.md
./txt2img/README.md
./visualqna/README.md
```

@poussa ?

Reply:
This is not a showstopper for the release.